### PR TITLE
Search left pane (dumb)

### PR DESCRIPTION
**Pull request by gabriel**
@MarcoPolo 
cc @keybase/react-hackers 

2016-06-15 at 9 55 am

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **user_search-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466009634575-8db4a174d9c9-23dfd2cc68d9/user_search-Normal-8db4a174d9c9.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466009634575-8db4a174d9c9-23dfd2cc68d9/user_search-Normal-23dfd2cc68d9.png)  
   user_search-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **user_search-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466010230915-50e8fcb90240-b4af1f712732/user_search-Normal-50e8fcb90240.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466010230915-50e8fcb90240-b4af1f712732/user_search-Normal-b4af1f712732.png)  
   user_search-Normal comparison

File: shared/search/user-search/render.desktop.js [hunk 1]
```diff
@@ -1,10 +1,215 @@
...
+    return (
+      <Box style={styles.container}>
+        <style>{realCSS}</style>
+        <Box style={styles.headerContainer}>
+          <Box style={styles.servicesContainer}>
+            <ServiceIcon
+              serviceName='keybase'
+              iconType='keybase-logo-mascot-only-dz-2-24'
+              selected={this.props.selectedService === 'keybase'}
+              onClickService={this.selectService}
```
Comment (MarcoPolo) [inline shared/search/user-search/render.desktop.js]: This should be a prop that's  passed in

File: shared/search/user-search/render.desktop.js [hunk 2]
```diff
@@ -1,10 +1,215 @@
...
+        {icon}
+        {alignedBody}
+        {extraInfo}
+      </Box>
+    </Box>
+  )
+}
+
+function ServiceIcon ({serviceName, iconType, selected, onClickService}: {serviceName: string, iconType: IconType, selected: boolean, onClickService: ServiceFn}) {
+  var iconStyles: Object = {
```
Comment (MarcoPolo) [inline shared/search/user-search/render.desktop.js]: Use `const`

Comment (MarcoPolo) [inline shared/search/user-search/render.desktop.js]: You don't need to type annotate this

File: shared/search/user-search/render.desktop.js [hunk 3]
```diff
@@ -1,10 +1,215 @@
...
+    borderRadius: 24,
+    minWidth: 48,
+    width: 48,
+    height: 48,
+    backgroundColor: null
```
Comment (MarcoPolo) [inline shared/search/user-search/render.desktop.js]: 🐑 `selected ? globalColors.blue4 : null`

File: shared/search/user-search/render.desktop.js [hunk 4]
```diff
@@ -1,10 +1,215 @@
...
+            <ServiceIcon
+              serviceName='pgp'
+              iconType='icon-pgp-key-24'
+              selected={this.props.selectedService === 'pgp'}
+              onClickService={this.selectService}
+              />
+          </Box>
+          <Input type='text' value={this.props.searchText} hintText={this.props.searchHintText} style={styles.input} underlineStyle={{display: 'none'}} />
+        </Box>
+        {this.props.results.map(r => <Result key={r.service + (r.icon || '') + r.username} result={r} searchText={this.props.searchText || ''} onClickResult={this.selectResult} />)}
```
Comment (MarcoPolo) [inline shared/search/user-search/render.desktop.js]: `onClickResult` should be passed in as a prop

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **user_search-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466023533019-3a0d0719d5ba-1e14b7a61ca4/user_search-Normal-3a0d0719d5ba.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466023533019-3a0d0719d5ba-1e14b7a61ca4/user_search-Normal-1e14b7a61ca4.png)  
   user_search-Normal comparison

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **user_search-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466033353863-09ad1d1059a4-2f6560e931fc/user_search-Normal-09ad1d1059a4.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466033353863-09ad1d1059a4-2f6560e931fc/user_search-Normal-2f6560e931fc.png)  
   user_search-Normal comparison

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**